R/setup.R

```diff
@@ -77,11 +77,27 @@ upload_enable <- as.logical(Sys.getenv(
   unset = get(params_obj_name)$toggle$upload_enable
 ))
 
-# If in a CI context, use the run note passed to the workflow. Otherwise, use
+# If in a CI context, use the run params passed to the workflow. Otherwise, use
 # the note included in params.yaml
 run_note <- as.character(
   Sys.getenv("WORKFLOW_RUN_NOTE", unset = get(params_obj_name)$run_note)
 )
+run_type <- as.character(
+  Sys.getenv("WORKFLOW_RUN_TYPE", unset = get(params_obj_name)$run_type)
+)
+
+# Must be one of the dedicated run types
+possible_run_types <- c(
+  "junk", "rejected", "test",
+  "baseline", "candidate", "final"
+)
+if (!run_type %in% possible_run_types) {
+  stop(
+    "Invalid run type '", run_type, "'. Must be one of: ",
+    paste0(possible_run_types, collapse = ", ")
+  )
+}
+rm(possible_run_types)
```

Comment:
Just dropping this so it doesn't stay in folks environment when they're working on the model.